plugins/terminal.koplugin/main.lua

```diff
@@ -349,9 +349,7 @@ function Terminal:generateInputDialog()
             {
             text = "⎚", --clear
             callback = function()
-                self.history = ""
-                self.input = {}
-                self.input_dialog:setInputText("$ ")
+                self:transmit("clear\n") -- this could happen during a running program (e.g. top)
```

Comment:
Is this `clear` a shell command you're sending, or some kind of escape code ?
If the former, what if it's not a shell, but eg "top" that gets it ?

Reply:
It is just a string that gets send. In top you get "wrong command", in nano you see "clear" ...

Reply:
May be sending Ctrl-L would work better in more contexts ?

Reply:
Sending Ctrl-L is not that easy. This would need some drill up of inputtext, which I have no time for now.
After quite some time thinking I found no elegant solution except dropping that button.
Btw we are a bit limited on esc sequences in the inputtext widget (e.g. bold ...)

So if nobody has a veto until tomorrow, I will drop it (comment it out ).